x/ibc/20-transfer/client/cli/tx.go

```diff
@@ -75,58 +82,68 @@ func GetTransferTxCmd(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 	cmd.Flags().Bool(FlagSource, false, "Pass flag for sending token from the source chain")
+	cmd.Flags().String(flags.FlagFrom, "", "key in local keystore to send from")
 	return cmd
 }
 
 // GetMsgRecvPacketCmd returns the command to create a MsgRecvTransferPacket transaction
 func GetMsgRecvPacketCmd(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "recv-packet [/path/to/packet-data.json] [/path/to/proof.json] [height]",
+		Use:   "recv-packet [sending-port-id] [sending-channel-id] [client-id]",
```

Comment:
this needs to be cleaned up

Reply:
Yup, it was a messy quick fix for the demo. We should think about how these commands will work.